stdlib/public/CMakeLists.txt

```diff
@@ -149,23 +149,42 @@ elseif(BOOTSTRAPPING_MODE STREQUAL "OFF")
 endif()
 
 if(SWIFT_SHOULD_BUILD_EMBEDDED_STDLIB)
-  set(EMBEDDED_STDLIB_TARGET_TRIPLES
-    # arch    module_name               target triple
-    "armv6    armv6-apple-none-macho    armv6-apple-none-macho"
-    "armv6m   armv6m-apple-none-macho   armv6m-apple-none-macho"
-    "armv7    armv7-apple-none-macho    armv7-apple-none-macho"
-    "armv7em  armv7em-apple-none-macho  armv7em-apple-none-macho"
-    "arm64    arm64-apple-none-macho    arm64-apple-none-macho"
-
-    # the following are all ELF targets
-    "armv6    armv6-none-none-eabi      armv6-none-none-eabi"
-    "armv6m   armv6m-none-none-eabi     armv6-none-none-eabi"
-    "armv7    armv7-none-none-eabi      armv7-none-none-eabi"
-    "armv7em  armv7em-none-none-eabi    armv7em-none-none-eabi"
-    "aarch64  aarch64-none-none-elf     aarch64-none-none-elf"
-    "riscv32  riscv32-none-none-eabi    riscv32-none-none-eabi"
-    "riscv64  riscv64-none-none-eabi    riscv64-none-none-eabi"
+  set(EMBEDDED_STDLIB_TARGET_TRIPLES)
+  if("ARM" IN_LIST LLVM_TARGETS_TO_BUILD)
```

Comment:
does this catch when we specify `--llvm-targets-to-build all`?

Reply:
It should. `all` is already the default, and it is transformed in https://github.com/apple/llvm-project/blob/94624bdf65b8ad61f9bf3948a5bf387c6fa0a967/llvm/CMakeLists.txt#L569-L571 to expand to the actual list. One never has to check for `"all"` explicitly.